src/main/scala/tilelink/Parameters.scala

```diff
@@ -366,6 +395,35 @@ object TLSlaveParameters {
       alwaysGrantsT = alwaysGrantsT,
       fifoId        = fifoId)
   }
+
+  def v2(
+    nodePath:      Seq[BaseNode]                = Seq(),
+    resources:     Seq[Resource]                = Seq(),
+    setName:       String,
```

Comment:
Again, this should probably be called name, and default to "".

Reply:
What about `Option[String] = None`?

Reply:
Not using an Option for this cause bug #2428 IMO